### PR TITLE
Update/issue11 suppress logging

### PR DESCRIPTION
**Pull request by kamontia**
ref : #11

```
+ ./qs -n 0..5 -f
Successfully rebased and updated refs/heads/master.   <-  show only result 
++ git diff 'HEAD^..HEAD' --name-only
```

### CODE REVIEW

File: script/test.sh [hunk 1]
```diff
@@ -65,8 +66,10 @@ test_run () {
 :
 if [ "prepare" != "$PREPARE" ]; then
   # test for `./qs -n 5 -f -d` and expected result value is 6
-  test_run 5 6
+  test_run 5 6 -d
   # test for `./qs -n 0..5 -f -d` and expected result value is 6
+  test_run 0..5 6 -d
+  # test for `./qs -n 0..5 -f ` and suppress logging
```
Comment (chaspy) [inline script/test.sh]: we can not test if verbose log suppressed...
but can test without debug flag 😄

File: script/test.sh [hunk 2]
```diff
@@ -39,9 +39,10 @@ teardown () {
 test_check () {
   NUM=$1
   EXPECTED_ADDED_FILE_NUM=$2
-  git log --oneline
-  ./$ExecComamnd -n $NUM -f -d
-  git log --oneline
+  OPTIONS=$3
+  #git log --oneline
```
Comment (chaspy) [inline script/test.sh]: is not this needed in any case?
if it is so, please remove.
if it is NOT, please issue the tasks.

Comment (kamontia) [reply script/test.sh]: OPTION value is needed to test this feature.

This test case shows debug log. OPTION value is set "-d"
```
*** START 0..5 ***
+ test_check 0..5 6 -d
+ NUM=0..5
+ EXPECTED_ADDED_FILE_NUM=6
+ OPTIONS=-d
+ ./qs -n 0..5 -f -d
INFO[0000] *** force update ***
INFO[0000] git rebaes HEAD~5 --exec=git commit --amend -m"Add file-5"
```

This test case suppresses debug log. OPTION value is set none.
```
*** START 0..5 ***
+ test_check 0..5 6
+ NUM=0..5
+ EXPECTED_ADDED_FILE_NUM=6
+ OPTIONS=
+ ./qs -n 0..5 -f
Successfully rebased and updated refs/heads/master.
```

Comment (chaspy) [reply script/test.sh]: I know that the OPTION variable is necessary,
I pointed out `#git log --oneline`
Sorry to misunderstand

Comment (kamontia) [reply script/test.sh]: I remove the lines.

**Comment by chaspy** [comment]
LGTM 👍

**Pull request merged**